extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -456,30 +474,53 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
                 }
             }
 
+            function _isDisabledRequest(xhr?: XMLHttpRequestInstrumented, request?: Request | string, init?: RequestInit) {
+                let isDisabled = false;
+                let theUrl:string = (!CoreUtils.isString(request) ? (request as Request).url || "" : request as string).toLowerCase();
+                let idx = theUrl.indexOf("?");
```

Comment:
Do you want to remove hash parameter as well here? for custom endpoint scenarios to be extra safe

Reply:
Sounds like a plan as they could have no '?' but a hash.